src/libsync/owncloudpropagator.h

```diff
@@ -227,7 +227,7 @@ class OWNCLOUDSYNC_EXPORT PropagateDirectory : public PropagatorJob {
 private slots:
     bool possiblyRunNextJob(PropagatorJob *next) {
         if (next->_state == NotYetStarted) {
-            connect(next, SIGNAL(finished(SyncFileItem::Status)), this, SLOT(slotSubJobFinished(SyncFileItem::Status)), Qt::QueuedConnection);
+            connect(next, SIGNAL(finished(SyncFileItem::Status)), this, SLOT(slotSubJobFinished(SyncFileItem::Status)));
```

Comment:
This makes that commit non-free of regression by itself, but I've spent too much time on this pull request already so I thought I might be OK.